Adding variable number for easier tracking

edkazcarlson opened the pull request:
Adding variable number for use by 3rd party applications with help from Andrew Au.

edkazcarlson committed:
Adding varibales for tracking

The changes let people track variables
better when making applications
that use runtime

edkazcarlson committed:
Merge remote-tracking branch 'upstream/master'

edkazcarlson committed:
removing todo comments

dnfadmin commented:
[![CLA assistant check](https://cla.dotnetfoundation.org/pull/badge/not_signed)](https://cla.dotnetfoundation.org/dotnet/runtime?pullRequest=39811) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/runtime?pullRequest=39811) before we can accept your contribution.<br/><br/>:x: edkazcarlson [sign now](https://cla.dotnetfoundation.org/dotnet/runtime?pullRequest=39811)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/runtime?pullRequest=39811) it.</sub>

edkazcarlson committed:
Added variable number for tracking

Added variable number to nativevarinfo so 
3rd party applications can more easily track variables.

edkazcarlson committed:
Merge remote-tracking branch 'upstream/master'

edkazcarlson committed:
Merge branch 'master' of https://github.com/edkazcarlson/runtime

edkazcarlson committed:
removed mistake from script
